src/archivematicaCommon/lib/elasticsearch/aip_mets_mapping.json

```diff
@@ -0,0 +1,739 @@
+                              "type": "string"
+                            },
+                            "ns4:agentType": {
+                              "type": "string"
+                            }
+                          }
+                        }
+                      }
+                    }
+                  }
+                },
+                "@ID": {
+                  "type": "string"
+                }
+              }
+            }
+          }
+        },
+        "@xmlns:ns0": {
+          "type": "string"
+        },
+        "@xmlns:ns2": {
+          "type": "string"
+        },
+        "@xmlns:ns5": {
+          "type": "string"
+        },
+        "@xmlns:ns4": {
+          "type": "string"
+        },
+        "ns0:dmdSec_dict_list": {
+          "properties": {
+            "ns0:mdWrap_dict_list": {
+              "properties": {
+                "@MDTYPE": {
+                  "type": "string"
+                },
+                "ns0:xmlData_dict_list": {
+                  "properties": {
+                    "ns2:dublincore_dict_list": {
+                      "properties": {
+                        "dc:contributor": {
+                          "type": "string"
+                        },
+                        "dc:source": {
+                          "type": "string"
+                        },
+                        "dc:coverage": {
+                          "type": "string"
+                        },
+                        "dc:provenance": {
+                          "type": "string"
+                        },
+                        "dc:creator": {
+                          "type": "string"
+                        },
+                        "dc:subject": {
+                          "type": "string"
+                        },
+                        "dc:identifier": {
+                          "type": "string"
+                        },
+                        "dc:language": {
+                          "type": "string"
+                        },
+                        "@xsi:schemaLocation": {
+                          "type": "string"
+                        },
+                        "dc:relation": {
+                          "type": "string"
+                        },
+                        "dc:date": {
+                          "type": "string"
+                        },
+                        "dc:publisher": {
+                          "type": "string"
+                        },
+                        "dc:format": {
+                          "type": "string"
+                        },
+                        "dc:title": {
+                          "type": "string"
+                        },
+                        "dc:description": {
+                          "type": "string"
+                        },
+                        "dc:rights": {
+                          "type": "string"
+                        }
```

Comment:
Curiously I did mark this as part of an AIC yet `isPartOf` didn't get indexed. I can manually add it though.

> Also, how does this handle dmdSecs that have non-DC metadata? We just don't index it?

Correct; when indexing objects you can always omit fields.